reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -1825,9 +1825,16 @@
 	 * @param <T> the produced type
 	 *
 	 * @return a {@link FluxProcessor} accepting publishers and producing T
+	 *
+	 * @deprecated to be removed in 3.6. In 3.5 the default value for {@code prefetch}
```

Comment:
the method itself is missing the `@Deprecated` annotation (only javadoc tag here)

Reply:
done